packages/embed/README.md

```diff
@@ -1,23 +1,57 @@
 # botframework-webchat-embed
 
-This package is the embed page hosting at https://webchat.botframework.com/.
+This package contains the source code for the embed page at https://webchat.botframework.com/.
 
-For transparency and completeness, we are maintaining the code here. This code is coupled to our servers. In order to host this page, you will need to decouple it or provide the same set of REST APIs.
+> For transparency and completeness, we are maintaining the code here. This code is coupled to our servers. In order to host this page on your own website, you will need to decouple it or provide the same set of REST APIs.
 
-## Build
+## Background
+
+Web Chat offers multiple levels of integration options. In this package, we focus on embedding `<iframe>` element in your web page.
+
+`<iframe>` is the simplest way to connect your website to Azure Bot Services. If you prefer more advanced features or customization options, please refer to [this migration guide](https://github.com/microsoft/botframework-webchat/blob/master/README.md#migrating-from-v3-to-v4-of-web-chat) for details.
+
+There are a few features unique to `<iframe>`:
+
+- Reading configurations from Azure Bot Services
+- Simplified version selection
+   - Based on the versions you selected (latest, specific major, or specific minor), we will serve Web Chat client with latest security fixes
+   - Please refer to [this section](#i-want-to-go-back-to-previous-versions-of-web-chat) to specifying version to use
+- Same experince as "Test in Web Chat" blade in Azure Portal
+
+## How to use
+
+To use the embed page, you will need the site secret from your Azure Bot Services page. Once you have the secret, add the following to your page.
+
+```html
+<iframe src="https://webchat.botframework.com/embed/your-bot-id?s=your-site-secret"></iframe>
+```
+
+For production use, we strongly prefer you to use token instead of secret. Please refer to [this article](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication) for how to generate token out of secret.
+
+```html
+<iframe src="https://webchat.botframework.com/embed/your-bot-id?t=your-site-token"></iframe>
+```
+
+Please look at [this table](#what-url-parameters-are-supported) for list of URL parameters supported.
+
+## Development
+
+### Building the page
+
+Follow steps below to build the embed page locally.
 
 1. `npm install`
 1. `npm run prepublishOnly`
-1. Publishes the build artifact at `/dist/index.html`, to our internal repository
+1. Publishes the build artifact at `/dist/index.html`
    1. Verifies the file size, it should be less than 30 KB
```

Comment:
```suggestion
   1. Verifies the file size. It should be less than 30 KB
```

Reply:
Weird, but Markdown use indentation of 3 spaces.

Reply:
Done.